regparser/tree/xml_parser/reg_text.py

```diff
@@ -229,16 +242,20 @@ def build_from_section(reg_part, section_xml):
         plain_sect_texts = [s[0] for s in section_texts]
         tagged_sect_texts = [s[1] for s in section_texts]
 
-        section_text = ' '.join([section_xml.text] + plain_sect_texts)
-        tagged_section_text = ' '.join([section_xml.text] + tagged_sect_texts)
         section_title = u"§ " + reg_part + "." + section_number
         if subject_text:
             section_title += " " + subject_text
 
-        sect_node = Node(
-            section_text, label=[reg_part, section_number],
-            title=section_title)
-        sect_node.tagged_text = tagged_section_text
+        # another workaround for Reg J
+        if reg_part == '1010':
+            sect_node = Node('', label=[reg_part, section_number], title=section_title)
+            sect_node.tagged_text = ''
```

Comment:
If I recall correctly, the section text is composed of all the non-labeled text in the section (with the assumption that this will be the intro text). Perhaps, then, this special case can be ignored by properly solving the previous issue, regarding unlabeled definitions.

Again, special cases like this are explicitly not in the main code base because they'd make the tool both difficult to maintain within CFPB and limit its application to the wider federal government.